### ISSUE TITLE
Strapi plugins installation fails when the path includes a whitespace

### ISSUE THREAD

**Issue by bobaekang** [strapi/strapi#3314]
**Informations**
- **Node.js version**: v10.15.3
- **NPM version**: v6.9.0
- **Strapi version**: v3.0.0-alpha.26.2
- **Database**: SQLite
- **Operating system**: Windows 10
- **(Optional) Link to your Project**: 

**What is the current behavior?**
Using `strapi new` fails to install all plugins with the following identical error message:

```
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\Bobae'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)       
```

Unfortunately, my Windows user profile includes a whitespace (`C:\Users\Bobae Kang\`), and it seems to be the cause.

**Steps to reproduce the problem**
Run `strapi new` command where a path to the global strapi installation includes a whitespace.

**What is the expected behavior?**
Plugins install successfully.

**Suggested solutions**
Strapi plugin installations handling a path including a whitespace.

**Comment by lauriejim** [comment]
Hello! Can you give me the full log of your `strapi new` please.

**Comment by lauriejim** [comment]
We fixed this issue in previous release.
Is it a fresh new app?
I just tested it and it works well.

**Comment by bobaekang** [comment]
@lauriejim 

Thank you for responding. Here is the full CLI output:
```
PS C:\Users\Bobae Kang\Documents\sandbox> strapi new test --quickstart


🚀 Starting to create your Strapi application.


✅ Connected to the database

🏗  Application generation:
√ Copy dashboard
‼ An error occurred during settings-manager plugin installation.
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\Bobae'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

‼ An error occurred during content-type-builder plugin installation.
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\Bobae'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

‼ An error occurred during content-manager plugin installation.
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\Bobae'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

‼ An error occurred during users-permissions plugin installation.
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\Bobae'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

‼ An error occurred during email plugin installation.
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\Bobae'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
>
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

‼ An error occurred during upload plugin installation.
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\Bobae'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

√ Link strapi dependency to the project.

👌 Your new application test is ready at C:\Users\Bobae Kang\Documents\sandbox\test.

⚡️ Change directory:
$ cd test

⚡️ Start application:
$ strapi start
```

And the path to the global strapi installation is "C:\Users\Bobae Kang\AppData\Roaming\npm".

Also, here is a screenshot of the admin panel that starts automatically--notice that the `plugins` section on the left navbar shows a message: `No plugins installed yet`:

![image](https://user-images.githubusercontent.com/22449454/58349242-63394500-7e28-11e9-8019-8c19ed240d48.png)

Lastly, I tried to `cd` into the created project and install individual plugins manually using `strapi install`, and the installation succeeded. On the contrary, trying it on the admin panel UI didn't seem to work.

**Comment by chaomediashaker** [comment]
Same issue here

**Comment by bobaekang** [comment]
**A quick update:**

While waiting to hear back about this issue, I was playing with `create-react-app` and found that npx does not like whitespaces in the path. This, in fact, seems to be a common pain point for Windows users. 

I then found [a workaround](https://github.com/zkat/npx/issues/100#issuecomment-413815760) that involves creating a symbolic link to the user directory without whitespace (in my case, `C:\Users\Bobae Kang\` -> `C:\Users\bobaekang\`)and changing the `npm config` settings to replace the paths including whitespace with those using the symbolic link (thus no whitespace). This solved my issue with `create-react-app`.

So I was hoping that maybe this can also help my issues with failing strapi plugin installation. Unfortunately, I'm seeing the exact same error messages. I tried re-installing strapi as well as `npm cache clean -f` to be sure, but still no luck... 😞
